traitsui/examples/demo/Advanced/Table_editor_with_live_search_and_cell_editor.py

```diff
@@ -328,36 +334,35 @@ def _get_source_files(self):
             if splitext(file_name)[1] in file_types
         ]
 
-    @property_depends_on('selected')
     def _get_selected_contents(self):
```

Comment:
don't we need a `@cached_property` here as well?

Reply:
Not necessarily, other than for efficiency, since the value is immutable.